lib/winexe.js

```diff
@@ -97,26 +146,38 @@ WinExe.prototype._getArgsForPsExec = function () {
  * @private
  */
 WinExe.prototype._exec = function (callback) {
+    var self = this;
+    var stdio = (this.isWindows) ? ['ignore', 'pipe', 'pipe'] : undefined;
```

Comment:
I made this optional because this stdio configuration wouldn't work in MacOS but appears to be required in Windows and PaExec